supported_platforms.md

```diff
@@ -0,0 +1,55 @@
+# Microsoft Azure IoT SDKs for Java
+
+This SDK is tested nightly on Windows 10, Ubuntu 1604, and on an Android emulator. For additional details
+for each tested platform, see the respective sections below. 
+
+While this SDK contains several different packages, each package supports the same platforms and JDKs.
+
+## Supported JDK versions
+
+Currently, this SDK supports JDK versions 8 and 11. This SDK may work with other versions, but there are no guarantees 
+that they will always work for those versions nor are there guarantees that we will fix bugs that are only present on
+those versions.
+
+
+## Windows 10
+
+Nightly test platform details:
+Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
+Java version: 1.8.0_292, vendor: AdoptOpenJDK, runtime: C:\hostedtoolcache\windows\Java_Adopt_jdk\8.0.292-10\x64\jre
+Default locale: en_US, platform encoding: Cp1252
+OS name: "windows server 2016", version: "10.0", arch: "amd64", family: "windows"
```

Comment:
I'd love to have a way for our nightly builds to automatically update these versions as the ADO agents we use update to later maven/JDK/windows versions, but I'll file that one under "Maybe some other time"

Reply:
this doesn't show up as expected in the readme (with newlines). You probably need to prefix each line with a bullet.

Reply:
Will do

Reply:
It looks like we test on Windows Server, but no mention of Win10 desktop client. We should we specify it somewhere